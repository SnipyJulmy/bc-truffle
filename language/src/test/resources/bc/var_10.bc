a = 421
define f() {
  return a
}
define g() {
  auto a
  a = 123
  return a
}
define h(a) {
  return a
}
define i() {
  a = 566
  return a
}
a
f()
a
g()
a
h(983621)
a
i()
a
halt
