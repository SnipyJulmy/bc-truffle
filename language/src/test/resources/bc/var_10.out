421
421
421
123
421
983621
421
566
566
